### PR TITLE
Add per establishment count of overdue ROPs to ROPs report

### PR DESCRIPTION
**Pull request by wheelsandcogs**

### CODE REVIEW

File: lib/routers/rops/index.js [hunk 1]
```diff
@@ -70,17 +70,38 @@ module.exports = () => {
...
     const year = req.year;
+    const now = (new Date()).toISOString();
 
     const ropsQuery = Establishment.relatedQuery('projects').count('id').whereRopsDue(year);
 
+    // get a list of all projects for the establishment with outstanding rops - ropsDeadline is generated by a SELECT
+    // in getRopsDeadline() so it cannot be used in a WHERE or a HAVING at this level
+    const outstandingRops = Project.query()
+      .select('id')
+      .getRopsDeadline(year)
```
Comment (lennym) [inline lib/routers/rops/index.js:84]: It's a little bit of 🐴 💩 💉  but is there any chance we can rename this `.selectRopsDeadline` to make it make a little more obvious what it does?

Comment (wheelsandcogs) [reply lib/routers/rops/index.js:84]: tbf I wanted to suggest that on Joe's original PR but thought I was being a bit anal so didn't.

Comment (wheelsandcogs) [reply lib/routers/rops/index.js:84]: https://github.com/UKHomeOffice/asl-schema/pull/486

Comment (wheelsandcogs) [reply lib/routers/rops/index.js]: Done

**Pull request merged**